imports/components/@enhancers/can-like/index.js

```diff
@@ -0,0 +1,90 @@
+
+import { Meteor } from "meteor/meteor";
+import React, { Component } from "react";
+import { graphql } from "react-apollo";
+import gql from "graphql-tag";
+
+import { liked as likedActions, modal } from "../../../data/store";
+import OnBoard from "../../people/accounts";
+
+const TOGGLE_LIKE_MUTATION = gql`
```

Comment:
at some point this sprint, we will need to make a fragment which pulls in all supported like types:

```graphql
id
... on Content {
  content { title, images { }, etc }
}
... on Group {
  name, photo
}
```

Then the mutation action can update the local store with the new like information. The update should be part of this enhancer, but the fragment shared between recently likes, my likes, and this.

So maybe `@enhancers/likes/fragment.js`, `@enhancers/likes/toggle.js`, `@enhancers/likes/publicLikes.js`, `@enhancers/likes/userLikes.js` ?

cc @edolyne